lib/tests/formulae.rb

```diff
@@ -82,39 +103,27 @@ def detect_formulae!(args:)
             "#{@argument} is not detected from GitHub Actions or a formula name!"
         end
 
-        # Use GitHub Actions variables for pull request jobs.
-        if ENV["GITHUB_BASE_REF"].present? && ENV["GITHUB_SHA"].present?
-          diff_start_sha1 = rev_parse("origin/#{ENV["GITHUB_BASE_REF"]}")
-          if diff_start_sha1.blank?
-            test git, "-C", repository, "fetch", "--depth=1",
-                 "origin", "+refs/heads/#{ENV["GITHUB_BASE_REF"]}",
-                 args: args
-            diff_start_sha1 = rev_parse("origin/#{ENV["GITHUB_BASE_REF"]}")
+        if ENV["GITHUB_REPOSITORY"].present?
+          # We're using GitHub Actions. Check the repo matches the tap we are using.
+          if tap&.full_name&.downcase == ENV["GITHUB_REPOSITORY"].downcase
```

Comment:
```suggestion
        if tap.present? && ENV["GITHUB_REPOSITORY"].present? && tap.full_name.downcase == ENV["GITHUB_REPOSITORY"].downcase
```

Would that work?

Reply:
I don't want the `else` clause if the tap doesn't match, so I'll need to duplicate the `ENV["GITHUB_REPOSITORY"].blank?` check in this case.

Reply:
Works for me 👍🏻